walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,140 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
```

Comment:
Can you add a javadoc for this class as well: https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required